.github/kubeval.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
It would be good to have an understanding of the copyright of this file. I am assuming you are the original author of it.

Reply:
added license header